doc/langref.html.in

```diff
@@ -9393,6 +9395,19 @@ fn List(comptime T: type) type {
       </p>
       {#header_close#}
 
+      {#header_open|@trap#}
+      <pre>{#syntax#}@trap() noreturn{#endsyntax#}</pre>
+      <p>
+      This function inserts a platform-specific trap/jam instruction which can be used to exit the program abnormally.
+      This may be implemented by explicitly emitting an invalid instruction which may cause an illegal instruction exception of some sort.
+      Unlike for {#syntax#}@breakpoint(){#endsyntax#}, execution does not continue after this point.
+      </p>
+      <p>
+      This function is only valid within function scope.
```

Comment:
```suggestion
      Outside function scope, this builtin causes a compile error.
```

Reply:
Hmm, I don't think it does? I didn't check for that but I'm guessing it has or should have the same behavior as `@breakpoint` in that regard:
```
$ cat x.zig
const std = @import("std");

const x = @breakpoint();

test {
    std.debug.print("{}\n", .{x});
}
$ zig test x.zig
Test [1/1] test_0... void
All 1 tests passed.
```
So I don't know if that should be a compile error, but maybe because it returns `noreturn` instead of `void`. So, should it not be valid within `test` scope either? Not sure.

Reply:
my suggested docs are correct; the compiler is wrong :-)